src/components/ProgressBar/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './ProgressBar.css';
+
+const ProgressBar = (props) => {
+  
+  let fillColor; 
+  if(props.percentage < 25) {
+    fillColor = 'Quarter1';
+  } else if (25 <= props.percentage && props.percentage < 50) {
+    fillColor = 'Quarter2';
+  } else if (50 <= props.percentage && props.percentage < 75) {
+    fillColor = 'Quarter3';
+  } else {
+    fillColor = 'Quarter4';
+  }
+
+  return (
+    <div className="ProgressBar">
+      <div
+        className={`ProgressBarFiller ProgressBarFill${fillColor}`}
+        style={{width: `${props.percentage}%`}}
+  >
+    {`${props.percentage}%`}
+    </div>
```

Comment:
Poor indenting from line 22 to 24

Reply:
Thanks for pointing this out @Mubangizi ... U got an eagle eye man...👁 
I've fixed it now

Reply:
Good stuff @StephenAraka